### ISSUE TITLE
Snapping sensation at time of injury

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9914]
**Preferred term label:**
Snapping sensation at time of injury

**Synonyms**
Sudden snapping, popping,

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Quadriceps tendon rupture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
this seems to be extremely specific to quadriceps tendon rupture or is there any case for making this into a mor general term, e.g., a modifier?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001203
- label: Snapping sensation at time of injury
- parents: [Pain characteristic (HP:0025280)](https://hpo.jax.org/app/browse/term/HP:0025280)
- pmid: 
- def: A popping or snapping sensation (noise or feeling) at the time of an injury.
- comment: Sudden snapping or popping at the time of an injury can be a sign of significant damage to muscles, tendons, ligaments, or bones.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9914
- synonyms:
